<launch>
    <arg name="event" default="iros2019"/>
    <arg name="size"/>
    <arg name="drum_calibration" default="0"/>

    <!-- robotis op3 manager -->
    <include file="$(find op3_manager)/launch/op3_manager.launch">
        <arg name="size" value="$(arg size)"/>
    </include>

    

    <node name="$(arg event)_node" pkg="events" type="$(arg event)_node.py" output="screen">
        <param name="event" type="str" value="$(arg event)" />
        <param name="size" type="string" value="$(arg size)" />
        <param name="drum_calibration" type="string" value="$(arg drum_calibration)"/>
    </node>
</launch>